src/core/annotation.js

```diff
@@ -1354,6 +1537,7 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
     if (isDict(fieldParent) && fieldParent.has("V")) {
       const fieldParentValue = fieldParent.get("V");
       if (isName(fieldParentValue)) {
+        this.parentRef = params.dict.getRaw("Parent");
```

Comment:
Given that `this.parentRef` might also be a `Dict` instead of a `Ref` that the name suggest, would `this.parent` perhaps be a better name?